build/cloud/07-deployment.yaml

```diff
@@ -26,6 +26,12 @@ spec:
             - containerPort: 10000
               name: cloudhub
               protocol: TCP
+            - containerPort: 10003
+              name: stream
+              protocol: TCP
+            - containerPort: 10004
+              name: tunnel
+              protocol: TCP
```

Comment:
I think we can delete the ports here, that's useless here, we have used `hostnetwork` here.

Reply:
Can the ports be removed here? cc @zhu733756 Please help take a look :)

Reply:
> Can the ports be removed here? cc @zhu733756 Please help take a look :)

I think it cannot be removed, and the related service type needs to use nodeport or loadblancer.  Therefore, these exposed services can be accessed by the edge nodes.

Reply:
@zhu733756 I just withdrew my submission. Do THE Stream and tunnel ports need to be reserved? Also should readinessGates be removed?

Reply:
> @zhu733756 I just withdrew my submission. Do THE Stream and tunnel ports need to be reserved? Also should readinessGates be removed?

Yeah. And I think you also need to expose nodeport for the cloudcore service.

Reply:
@zhu733756 Ok, I need to think about it. There is another problem. I have seen that Cloudcore has functions related to leader-election in the 1.6 version of the code, but the 1.7 version has deleted all the relevant codes. Do you know the reason?

Reply:
> @zhu733756 Ok, I need to think about it. There is another problem. I have seen that Cloudcore has functions related to leader-election in the 1.6 version of the code, but the 1.7 version has deleted all the relevant codes. Do you know the reason?

KubeEdge support ha-cloudcore now . See #2867

Reply:
Okey，thanks.

Reply:
@allenleu All the leader selection related code can be removed. :)

Reply:
Also you can take a look for #3304, we can use helm to deploy the cloudcore. @allenleu